pkg/util/shell/shell.go

```diff
@@ -90,7 +90,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 		prefix = "SET "
 		suffix = "\n"
 		if pathVar {
-			suffix = ";%PATH%" + suffix
+			suffix = ";%PATH%;" + suffix
```

Comment:
That is what suggested here https://github.com/minishift/minishift/issues/866#issuecomment-300722809 and seems working.